### DISCUSSION TITLE
Twitter Bot - Links Helper

### DISCUSSION THREAD

**Discussion by keremh** [acikkaynak/deprem-yardim-projesi#33] [Feedback]
Selamlar,

Twitter'da tweet'lere mentionlandığı anda şu anda yayında olan ve insanlara duyurmaya çalışılan tüm uygulamaları içeriğiyle birlikte tweet atacak.

Tek bir görevi olacak; yayındaki tüm uygulamaları mentionlandığı kişiye duyurmak. Çok faydalı olabileceğini düşünüyorum.  Ne dersiniz? 
 

Daha önce tweet stream bot tecrübem var, faydasıyla ilgili önerilerinizi duymak istedim.

**Comment by ramazansancar** [comment]
Merhaba @keremh,
Tartışma'ı Feedback kategorisine taşıdım.
Fikrinizi Issues olarak [buraya](https://github.com/acikkaynak/afet-yardim-twitter-bot) ve [buraya](https://github.com/acikkaynak/deprem-yardim-twitter-scraper) taşıyabilirsiniz.

**Reply by keremh** [reply]
teşekkürler yönlendirme için @ramazansancar
